Docs small updates (#10635)
hashicorp/packer: website/content/partials/guides/hcl2-beta-note.mdx

> -> **Note:** Starting from version **1.5.0** HCL2 templates were first introduced as a beta feature into Packer. As of **1.7.0**,
> HCL2 support is no longer in beta, and is the preferred way to write Packer
> configuration.

Reviewer: ```suggestion
-> **Note:** Packer version **1.5.0** introduced support for HCL2 templates as a beta feature. As of version **1.7.0**, HCL2 support is no longer in beta and is the preferred way to write Packer configuration(s).
```

What do you think of this suggestion ?

Author: I like it!
